src/main/java/xyz/groundx/caver_ext_kas/kas/tokenhistory/TokenHistory.java

```diff
@@ -102,6 +118,18 @@ public PageableTransfers getTransferHistory(List<Integer> presets) throws ApiExc
     /**
      * Gets transfer history list.<br>
      * GET /v2/transfer
+     *
+     * <pre>Example:
+     * {@code
+     * int preset = 1;
+     *
+     * TokenHistoryQueryOptions options = new TokenHistoryQueryOptions();
+     * options.setKind("ft");
```

Comment:
Please modify a setter with enum.

Reply:
@sirano11 
Thanks for your feedback. I'll update this PR soon.